src/collections/_documentation/development/sdk-dev/event-payloads/index.md

```diff
@@ -54,22 +54,21 @@ The following attributes are required for all events.
 
 `timestamp`
 
-: Indicates when the logging record was created (in the Sentry SDK). The Sentry
-  server assumes the time is in UTC. The timestamp should be in ISO 8601 format,
-  without a timezone.
+: Indicates when the logging record was created (in the Sentry SDK). The format
```

Comment:
Since we're updating this paragraph already, should we rename this to "event" and put some more emphasis on that it was created in the SDK?

```suggestion
: Indicates when the vevent was created in the Sentry SDK. The format
```

Reply:
Applied this change locally to also rewrap the paragraph as the line length changed.